# KNM
Korean Nutrition Model 
![Graphical abstract](https://github.com/user-attachments/assets/1865c27b-f615-4523-9593-204145160baa)


Nutrigenomics-based in silico analysis used in this study. 


(i) Construction of the Korean Nutrition Model (KNM): Missing carbohydrate and vitamin data were curated using a decision algorithm integrating MFDS, FooDB, and literature sources. 


(ii) Development of individual gut microbiome models: Gut microbial taxonomic profiles were generated from 16S rRNA sequencing data and processed with QIIME2 and SILVA databases. 


(iii) Integration of genome-scale metabolic reactions: Gut microbial metabolic functions were mapped using KEGG and AGORA databases. 


(iv) Nutrigenomics-based in silico analysis for diet-dependent gut microbiome modeling: Using MICOM, microbial growth rates and metabolic fluxes were predicted based on diet-specific metabolic inputs. 



To perform Korean nutrigenomics-based in silico analysis, users can donwload Korean Nutrition Model v.1.2.qza from here. 



References


mSystems Volume 5, Issue 1 (2020)


Nature Biotechnology volume 35, pages81â€“89 (2017)

